### PR TITLE
Scala Steward PRs 5660 - 5669 [BA-6518]

### PR DESCRIPTION
**Pull request by mcovarr**
Another 10, only review if Chris's stuff blows up.

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -109,8 +109,8 @@ object Dependencies {
   private val tikaV = "1.20"
   private val typesafeConfigV = "1.3.3"
   private val workbenchGoogleV = "0.15-2fc79a3"
-  private val workbenchModelV = "0.10-6800f3a"
-  private val workbenchUtilV = "0.3-f3ce961"
+  private val workbenchModelV = "0.14-27810079-SNAP"
+  private val workbenchUtilV = "0.6-27810079-SNAP"
```
Comment (cjllanwarne) [inline project/Dependencies.scala:113]: After standup I assumed you were just going to leave these unchanged. Is there method on choosing these specific versions? Should we be worried that they're still SNAPs?

Comment (mcovarr) [reply project/Dependencies.scala:113]: there are 3 of these, these 2 I updated b/c they weren't obviously breaking anything

**Comment by cjllanwarne** [comment]
Needs rebase onto the newly merged 30-39 set to make sure we don't have novel import collisions

**Comment by cjllanwarne** [comment]
@kshakir Created as:
* https://broadworkbench.atlassian.net/browse/BA-6555
* https://broadworkbench.atlassian.net/browse/BA-6556

**Pull request merged**